DataFormats/CTPPSDigi/interface/CTPPSPixelDataError.h

```diff
@@ -0,0 +1,55 @@
+#ifndef DataFormats_CTPPSPixelDataError_h
+#define DataFormats_CTPPSPixelDataError_h
+
+//---------------------------------------------------------------------------
+//!  \class CTPPSPixelDataError
+//!  \brief Pixel error -- collection of errors 
+//!
+//!  Class storing info about pixel errors
+//!  
+//---------------------------------------------------------------------------
+
+#include "FWCore/Utilities/interface/typedefs.h"
+
+#include <string>
+
+class CTPPSPixelDataError {
+  public:
+
+    /// Default constructor
+    CTPPSPixelDataError();
+    /// Constructor for 32-bit error word
+    CTPPSPixelDataError(uint32_t errorWord32, const int errorType, int fedId);
+    /// Constructor with 64-bit error word and type included (header or trailer word)
+    CTPPSPixelDataError(uint64_t errorWord64, const int errorType, int fedId);
+    /// Destructor
+    ~CTPPSPixelDataError();
+
+    void setWord32(uint32_t errorWord32);		// function to allow user to input the error word (if 32-bit) after instantiation
+    void setWord64(uint64_t errorWord64);		// function to allow user to input the error word (if 64-bit) after instantiation
+    void setType(int errorType);			// function to allow user to input the error type after instantiation
+    void setFedId(int fedId);			        // function to allow user to input the fedID after instantiation
+    void setMessage();				        // function to create an error message based on errorType

```

Comment:
the class is simple enough that these modifiers should not be necessary.
Notably, none of these are used in the code that creates CTPPSPixelDataError objects.
Please remove.

Reply:
Done. Just left for the "message" method.